packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -576,23 +654,37 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
     }
 
     private getTimeForX(x: number): bigint {
-        if (!this.chartRef.current?.chartInstance) {
-            return BigInt(0);
+        if (this.isBarPlot) {
+            if (!this.chartRef.current) {
+                return BigInt(0);
+            }
+        } else {
+            if (!this.chartRef.current?.chartInstance) {
+                return BigInt(0);
+            }
         }
         const offset = this.props.viewRange.getOffset() ?? BigInt(0);
         const duration = this.props.viewRange.getDuration();
+        const chartWidth = this.getChartWidth();
```

Comment:
To be theoretically robust, we should check for 0 to avoid div by zero below.
But in practice I think this method is only called from mouse events which can't happen if chart width is 0...

Reply:
Added the check.